modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/ScriptManagerBridge.cs

```diff
@@ -1253,16 +1258,16 @@ public void Dispose()
 
                 // There's no recursion here, so it's ok to go with a big enough number for most cases
                 // stackMaxSize = stackMaxLength * sizeof(godotsharp_property_def_val_pair)
```

Comment:
I guess the comment should be updated as well.

```suggestion
                // StackMaxSize = StackMaxLength * sizeof(godotsharp_property_def_val_pair)
```